spec/features/admin/variant_overrides_spec.rb

```diff
@@ -199,6 +199,9 @@
 
             expect do
               click_button 'Save Changes'
+
+              # We need to wait_until because the save action is not fast enough for the have_content matcher
+              wait_until { page.find("#status-message").text != "Saving..." }
```

Comment:
the only thing I know is that this method was long discouraged and it seems it got removed in Capybara 2.0 according to  https://groups.google.com/forum/#!topic/ruby-capybara/qQYWpQb9FzY, but to my surprise, we added it back in

https://github.com/openfoodfoundation/openfoodnetwork/blob/12000d2d4482936794f040d50dfaf5c6da5239f7/spec/support/request/web_helper.rb#L90-L96

This solution is rather similar to https://thoughtbot.com/blog/automatically-wait-for-ajax-with-capybara. It seems to me that the root cause might be the same, so :+1: